examples/solvers/chambolle_pock_tomography_tgv.py

```diff
@@ -0,0 +1,111 @@
+"""Total generalized variation tomography using the Chambolle-Pock solver.
+
+Solves the optimization problem
+
+    min_x ||A(x) - g||_2^2 + TGV_2(x)
+
+Where ``A`` is a parallel beam forward projector and ``g`` is given noisy data.
+TGV_2 is the second order total generalized variation of x, defined as
+
+    TGV_2(x) = min_y lam_1 ||grad(x) - y||_1 + lam_2 ||eps(y)||_1
```

Comment:
eps is actually first derivatives, my fault. 1 norm is simply sum of elements as usual